src/coredump/coredumpctl.c

```diff
@@ -933,6 +995,11 @@ int main(int argc, char *argv[]) {
                 assert_not_reached("Shouldn't be here");
         }
 
+        if (units_active > 0)
+                printf("%s-- Warning: %d systemd-coredump@.service %s, output may be incomplete.%s\n",
```

Comment:
Changed to "Notice". I still keep it red because otherwise it's too easy to miss. It's very likely to want to look at the last coredump, so we really want to point out if any are still being processed.